mmd_tools/__init__.py

```diff
@@ -144,7 +144,18 @@ def register():
         bpy.types.VIEW3D_MT_armature_add.append(menu_func_armature)
         #bpy.context.preferences.filepaths.use_scripts_auto_execute = True
 
+    try:
+        from mmd_tools.m17n import translation_dict
+        bpy.app.translations.register(bl_info['name'], translation_dict)
+    except:
+        pass
+
 def unregister():
+    try:
+        bpy.app.translations.unregister(bl_info['name'])
+    except:
+        pass
```

Comment:
I think it is better to remove `try...except...` unless we know which errors could be thrown by Blender.

Reply:
Thanks for your review. I removed the unnecessary try-except.